src/viewer/Viewer.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from 'react'
+import classNames from 'classnames'
+
+import ViewerToolbar from './ViewerToolbar'
+import ImageViewer from './ImageViewer'
+import NoViewer from './NoViewer'
+
+import styles from './viewer'
+
+const KEY_CODE_LEFT = 37
+const KEY_CODE_RIGHT = 39
+
+export default class Viewer extends Component {
+  state = {
+    controlsHidden: false
+  }
+
+  showControls = () => {
+    this.setState({ controlsHidden: false })
+    document.removeEventListener('mousemove', this.showControls)
+  }
+  hideControls = () => {
+    this.setState({ controlsHidden: true })
+    document.addEventListener('mousemove', this.showControls)
+  }
+
+  componentDidMount() {
+    document.addEventListener('keydown', this.onKeyDown, false)
```

Comment:
I think it should listen to `keyup` rather than `keydown` or we need to be sure that keeping keydown pressed renders the correct behavior: fast browsing files.

Reply:
It works better with `keyup` :+1: